README.rst

```diff
@@ -24,6 +24,37 @@ allowing us to inject it into the logs.
 * Documentation: https://django-guid.readthedocs.io
 
 
+Example
+-------
+
+Using ``demoproj`` as an example, all the log messages **without** ``django-guid`` would look like this:
+
+.. code-block:: bash
+
+    INFO 2020-01-14 12:28:42,194 django_guid.middleware No Correlation-ID found in the header. Added Correlation-ID: 97c304252fd14b25b72d6aee31565843
+    INFO 2020-01-14 12:28:42,353 demoproj.views This is a DRF view log, and should have a GUID.
+    INFO 2020-01-14 12:28:42,354 demoproj.services.useless_file Some warning in a function
+
+
+With ``django-guid`` every log message has a GUID attached to it(``97c304252fd14b25b72d6aee31565843``),
+through the entire stack:
+
+.. code-block:: bash
+
+    INFO 2020-01-14 12:28:42,194 [None] django_guid.middleware No Correlation-ID found in the header. Added Correlation-ID: 97c304252fd14b25b72d6aee31565843
```

Comment:
Could maybe add an extended example of this, with multiple GUIDs to illustrate the effect it would have when, e.g., logs span several endpoint, or multiple things are happening in parallel. Don't have to do it now though.

Reply:
I could add this as a extended example in the `readthedocs`. I'll do that, cheers!

Reply:
Added. Please have a look :)

Reply:
[Link](https://github.com/JonasKs/django-guid/blob/return_header/docs/extended_example.rst)

Also added this to the index and README:
```
For multiple requests at the same time over multiple threads, see the :ref:`extended_example`.
```